tools/github-action-reporter.js

```diff
@@ -52,7 +53,9 @@ function printFailure(failure) {
  * @param {Record<string, string>} options
  */
 function print(command, message = "", options = {}) {
-  let optionsString = new URLSearchParams(options).toString();
+  let optionsString = new URLSearchParams(options)
+    .toString()
+    .replace(/&/g, "%2C");
```

Comment:
I don't think `%2C` will work as a separator 👀

Reply:
I'm trying to follow what @actions/core did :thinking: 
Should I try `,` instead?

Reply:
Wait. Did you mean to not run replace again and again, and instead run it once on final string?

Reply:
I think `,` is what the previous code did, so yes.

Reply:
No, just here, because the purpose here is to use `,` as a separator between key-values.

Reply:
I'm gonna try once more... I should probably be sleeping on it :stuck_out_tongue:

Reply:
/me gives up for tonight

Reply:
Good night!

Reply:
Feel free to edit it while I sleep :smile_cat:

Reply:
Had an unrelated error. I refactored to use `Array.prototype.flatMap`, which doesn't work on Node v10 (which is used in our GitHub workflow), and I was debugging ~15 wrong commits.
Also, had to unescape `%2F` to `/`.